timescaledb/quick-start/dotnet.md

```diff
@@ -0,0 +1,493 @@
+Enables scalable inserts and complex queries for time-series data
+Finished dropping table (if existed)
+Finished creating the sensors table
+Number of rows inserted=1
+Number of rows inserted=1
+Number of rows inserted=1
+Number of rows inserted=1
+Dropped sensor_data table if it existed
+Created sensor_data table to store time-series data
+Converted the sensor_data table into a TimescaleDB hypertable!
+Number of rows inserted=1441
+Number of rows inserted=1441
+Number of rows inserted=1441
+Number of rows inserted=1441
+```
+
+Congratulations! You have successfully created and inserted time-series data into your hypertable. The last step to is to execute your first `time_bucket()` query against the hypertable data.
+
+## Query TimescaleDB [](query-timescaledb)
+As a final step of this Quick Start, use the time_bucket function to analyze the average CPU reading in 5 minute buckets of time. This query returns all data for all sensors as currently written. However, this method could be refactored to receive inputs that further filter data using a WHERE clause.
+
+After executing the query, iterate the results using the `NpgsqlDataReader` and print the results to the console.
+
+### Procedure: Querying TimescaleDB
+
+1. Create the final method at the bottom of the `TimescaleHelper` class for querying time-series data using the `time_bucket()` function provided by TimescaleDB.
+
+```csharp
+        //
+        // Procedure - Query TimescaleDB
+        // With time-series data inserted, run a 'time_bucket()' query
+        // on the data in order to aggregate our 1-minute cpu data into buckets
+        // of 5-minute averages.
+        public void RunQueryExample(){
+            string sql = @"
+                SELECT sensor_id, time_bucket('5 minutes', time) AS five_min, avg(cpu)
+                FROM sensor_data
+                    INNER JOIN sensors ON sensors.id = sensor_data.sensor_id
+                GROUP BY sensor_id, five_min
+                ORDER BY sensor_id, five_min DESC;";
+
+            var conn = getConnection();
+            using(var cmd = new NpgsqlCommand(sql, conn))
+            {
+                using(NpgsqlDataReader rdr = cmd.ExecuteReader()) {
+                    while (rdr.Read()) Console.WriteLine($"{rdr.GetDouble(0)} - {rdr.GetTimeStamp(1)} - {rdr.GetDouble(2)}");
+                }
+            }
+
+            conn.Close();
+        }
+```
+
+2. Call this method in the `Main` program directly after the `ts.InsertData();` reference.
+
+```csharp
+            // Procedure - Query TimescaleDB
+            // Query the data using the Timescale time_bucket() function
+            ts.RunQueryExample();
+```
+
+3. Save and run the application again. As before, if you execute all of the methods in the `Main` program, your output should look similar to this. **Note**, the values of the output will be different than what is shown because we used the `random()` function to generate the values.
+
```

Comment:
```suggestion
1. Save and run the application again. As before, if you execute all of the methods in the `Main` program, your output should look similar to this. The values of the output will be different because we used the `random()` function to generate them:
```